# Studying NGS data analysis
The repository is for studying the methods of NGS data analysis.
***
##Software & Algorithm
####[RaceID](https://github.com/dgrun/RaceID)  
######Infor:  
RaceID is an algorithm for the identification of rare and abundant cell types from single cell transcriptome data. The method is based on transcript counts obtained with unique molecular identifies.  
######Paper:  
Dominic Gr√ºn, Anna Lyubimova, Lennart Kester, Kay Wiebrands, Onur Basak, Nobuo Sasaki, Hans Clevers & Alexander van Oudenaarden (2015) Single-cell mRNA sequencing reveals rare intestinal cell types. Nature DOI:10.1038/nature14966

***
####[DaPars](https://github.com/ZhengXia/DaPars)  
######Infor:  
DaPars is the first de novo tool that directly infers the dynamic alternative polyadenylation (APA) usage by comparing standard RNA-seq. Given the annotated gene model, DaPars can infer the de novo proximal APA sites as well as the long and short 3'UTR expression levels. Finally, the dynamic APA usages between two conditions will be identified.  
######Paper:  
Masamha, C.P., Xia, Z., Yang, J., Albrecht, T.R., Li, M., Shyu, A., Li, W., Wagner, E.J. 2014. CFIm25 links Alternative Polyadenylation to Glioblastoma Tumor Suppression. Nature, 510:412-416.  
Xia, Z., Donehower, L.A., Wheeler, D.A., Cooper, T.A., Neilson, J.R., Wagner E.J., Li, W. 2014. Dynamic Analyses of Alternative Polyadenylation from RNA-Seq Reveal 3'-UTR Landscape Across 7 Tumor Types. Nature Communications, 5:5274.
***